py/selenium/webdriver/remote/remote_connection.py

```diff
@@ -471,6 +471,9 @@ def _request(self, method, url, body=None):
             request.add_header('Accept', 'application/json')
             request.add_header('Content-Type', 'application/json;charset=UTF-8')
 
+            base64string = base64.b64encode('%s:%s' % (parsed_url.username, parsed_url.password))
```

Comment:
This will always add the authorization header to the request object. Is this the right scope for these two lines? If username/password are not defined, it will encode 'Basic :'